Dockerfile.glibc

```diff
@@ -19,15 +19,16 @@ RUN rustup component add rust-src
 # builder stage
 WORKDIR /iroha
 COPY . .
-RUN mold --run cargo build --target x86_64-unknown-linux-gnu --profile deploy
+ARG PROFILE="deploy"
+ARG RUSTFLAGS=""
+RUN RUSTFLAGS="-C target-feature=+crt-static ${RUSTFLAGS}" mold --run cargo build --target x86_64-unknown-linux-gnu --profile "${PROFILE}"
 
 # final image
 FROM alpine:3.18
 
-ENV  GLIBC_REPO=https://github.com/sgerrand/alpine-pkg-glibc
-ENV  GLIBC_VERSION=2.35-r1
+ARG PROFILE="deploy"
```

Comment:
redefined here?

Reply:
Not redefined, but defined again in the different image

Reply:
I have to do it due to multistage build.

Reply:
> An ARG instruction goes out of scope at the end of the build stage where it was defined. To use an argument in multiple stages, each stage must include the ARG instruction.

From the [reference](https://docs.docker.com/engine/reference/builder/#arg).